includes/Checker/Preparations/Force_Single_Plugin_Preparation.php

```diff
@@ -82,7 +82,14 @@ public function prepare() {
 	public function filter_active_plugins( $active_plugins ) {
 		if ( is_array( $active_plugins ) && in_array( $this->plugin_basename, $active_plugins, true ) ) {
 
-			$plugin_base_file = plugin_basename( WP_PLUGIN_CHECK_MAIN_FILE );
+			if ( defined( 'WP_PLUGIN_CHECK_MAIN_FILE' ) ) {
+				$plugin_check_file = WP_PLUGIN_CHECK_MAIN_FILE;
+			} else {
+				$plugins_dir       = defined( 'WP_PLUGIN_DIR' ) ? WP_PLUGIN_DIR : WP_CONTENT_DIR . '/plugins';
```

Comment:
Why would `WP_PLUGIN_DIR` not be defined?

Reply:
@spacedmonkey the preparations are run when the `object-cache.php` file is loaded. This happens when the `WP_CONTENT_DIR` has been defined, but before the default `WP_PLUGIN_DIR` is, so it may not be set at this point in the load process.